### PR TITLE
Fix nightly build gpu

### PR DESCRIPTION
**Pull request by stu1130**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Comment by stu1130** [comment]
passed the ./gradlew build command

File: dlr/dlr-engine/src/test/java/ai/djl/dlr/integraion/DlrTest.java [hunk 1]
```diff
@@ -58,6 +59,7 @@ public void testDlr() throws ModelException, IOException, TranslateException {
                         .optFilter("os", os)
                         .optTranslator(translator)
                         .optEngine("DLR")
+                        .optDevice(Device.cpu()) // GPU is not supported
```
Comment (frankfliu) [inline dlr/dlr-engine/src/test/java/ai/djl/dlr/integraion/DlrTest.java:62]: This looks like a bug in DlrModeZoo, please add a FIXME here.

**Review by zachgk — APPROVED** [review]

**Review by frankfliu — CHANGES_REQUESTED** [review]
Please hold on.

**Comment by stu1130** [comment]
close the PR in favor of https://github.com/awslabs/djl/pull/403
